sync/src/main/java/tech/pegasys/artemis/sync/SyncingStatus.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.sync;
+
+public class SyncingStatus {
+  private final boolean is_syncing;
+  private final SyncStatus syncStatus;
+
+  public SyncingStatus(final boolean is_syncing, final SyncStatus syncStatus) {
+    this.is_syncing = is_syncing;
+    this.syncStatus = syncStatus;
+  }
+
+  public SyncStatus getSync_status() {
+    return syncStatus;
+  }
+
+  public boolean isIs_syncing() {
+    return is_syncing;
+  }
```

Comment:
really not a fan of isIs_syncing..
Maybe if the flag is called 'syncing' and then the is function could be isSyncing...

Reply:
made public final with the proper names